### ISSUE TITLE
Lot of noise from run console in iOS

### ISSUE THREAD

**Issue by xster** [flutter/flutter#8981]
Had a small app running in https://github.com/xster/flutter-plugin-share

```
Reload done: 2 of 402 libraries needed reloading.
discovered extensions
subsystem: com.apple.social, category: core, enable_level: 0, persist_level: 0, default_ttl: 7, info_ttl: 7, debug_ttl: 0, generate_symptoms: 0, enable_oversize: 0, privacy_setting: 0, enable_private_data: 0
<private> adding listener `<private>` for service: iMessage
<private> creating monitor for service: iMessage
Updating GS Capabilities
Device has regionInfo: <private>
               Supports SMS: YES
               Supports MMS: YES
             MMS Configured: YES
                         GT: NO
            FTA Force Allow: NO
                Supports FT: YES
               Supports FTA: YES
          Supports iMessage: YES
              Supports WiFi: YES
             Supports 3G FT: YES
              Supports WLAN: NO
         Supports Cell Data: YES
      Supports Front Camera: YES
       Supports Back Camera: YES
           Supports Handoff: YES
         Supports Tethering: YES
          Supports ApplePay: YES
       Celluar Queries Only: NO
               SIM Inserted: NO
Failed to read values in CFPrefsPlistSource<0x17410dda0> (Domain: com.apple.MobileSMS, User: kCFPreferencesCurrentUser, ByHost: No, Container: kCFPreferencesNoContainer): accessing preferences outside an application's container requires user-preference-read or file-read-data sandbox access, detaching from cfprefsd
subsystem: com.apple.SystemConfiguration, category: SCPreferences, enable_level: 0, persist_level: 0, default_ttl: 0, info_ttl: 0, debug_ttl: 0, generate_symptoms: 0, enable_oversize: 0, privacy_setting: 2, enable_private_data: 0
"The connection to ACDAccountStore was invalidated."
"The connection to ACDAccountStore was invalidated."
"The connection to ACDAccountStore was invalidated."
"The connection to ACDAccountStore was invalidated."
subsystem: com.apple.sharing, category: AirDrop, enable_level: 0, persist_level: 0, default_ttl: 3, info_ttl: 0, debug_ttl: 0, generate_symptoms: 0, enable_oversize: 0, privacy_setting: 2, enable_private_data: 0
Registering <private> for carrier changes
Unregistering for comm center launch
Registering for comm center launch
Registering <private> for lockdown changes
subsystem: com.apple.SystemConfiguration, category: SCDynamicStore, enable_level: 0, persist_level: 0, default_ttl: 0, info_ttl: 0, debug_ttl: 0, generate_symptoms: 0, enable_oversize: 0, privacy_setting: 2, enable_private_data: 0
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=0> with style: TurnOn; activationCount: 0 -> 1; styleActivationCount: 0 -> 1
feedback engine <_UIFeedbackSystemSoundEngine: 0x1740b48e0: state=4, numberOfClients=0, prewarmCount=0, _isSuspended=0> state changed: Running -> Inactive
subsystem: com.apple.sharing, category: Browser, enable_level: 0, persist_level: 0, default_ttl: 3, info_ttl: 0, debug_ttl: 0, generate_symptoms: 0, enable_oversize: 0, privacy_setting: 2, enable_private_data: 0
SFOperationCallBack (<private>, event = Information, results = <private>)
SFBrowserCallBack (node = <SFNode 0x1702ebd80>{domain = AirDrop})
(
subsystem: com.apple.amp.AirTraffic, category: StoreDownloads-Oversize, enable_level: 0, persist_level: 0, default_ttl: 0, info_ttl: 0, debug_ttl: 0, generate_symptoms: 0, enable_oversize: 1, privacy_setting: 0, enable_private_data: 0
subsystem: com.apple.amp.AirTraffic, category: iTunesSync-Oversize, enable_level: 0, persist_level: 0, default_ttl: 0, info_ttl: 0, debug_ttl: 0, generate_symptoms: 0, enable_oversize: 1, privacy_setting: 0, enable_private_data: 0
subsystem: com.apple.amp.AirTraffic, category: SyncBundle-Oversize, enable_level: 0, persist_level: 0, default_ttl: 0, info_ttl: 0, debug_ttl: 0, generate_symptoms: 0, enable_oversize: 1, privacy_setting: 0, enable_private_data: 0
subsystem: com.apple.amp.AirTraffic, category: Framework-Oversize, enable_level: 0, persist_level: 0, default_ttl: 0, info_ttl: 0, debug_ttl: 0, generate_symptoms: 0, enable_oversize: 1, privacy_setting: 0, enable_private_data: 0
subsystem: com.apple.amp.HomeSharing, category: Connections-Oversize, enable_level: 0, persist_level: 0, default_ttl: 0, info_ttl: 0, debug_ttl: 0, generate_symptoms: 0, enable_oversize: 1, privacy_setting: 0, enable_private_data: 0
subsystem: com.apple.amp.HomeSharing, category: XPC-Oversize, enable_level: 0, persist_level: 0, default_ttl: 0, info_ttl: 0, debug_ttl: 0, generate_symptoms: 0, enable_oversize: 1, privacy_setting: 0, enable_private_data: 0
subsystem: com.apple.UIKit, category: ViewServicesRDAR22970009, enable_level: 0, persist_level: 0, default_ttl: 0, info_ttl: 0, debug_ttl: 0, generate_symptoms: 0, enable_oversize: 1, privacy_setting: 2, enable_private_data: 0
Connection to sharingd became invalid
Connection to sharingd became invalid
if we're in the real pre-commit handler we can't actually add any new fences due to CA restriction
errors encountered while discovering extensions
Connection to sharingd became invalid
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 0; styleActivationCount: 1 -> 0
feedback engine <_UIFeedbackSystemSoundEngine: 0x1740b48e0: state=0, numberOfClients=1, prewarmCount=0, _isSuspended=0> state changed: Inactive -> Activating
feedback engine <_UIFeedbackSystemSoundEngine: 0x1740b48e0: state=3, numberOfClients=1, prewarmCount=0, _isSuspended=0> state changed: Activating -> Running
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x17410b370; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269587.160361
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x170112090; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269587.290728
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x1741119d0; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269587.393358
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x174111e50; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269587.576085
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x1701125a0; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269587.706443
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x174111ee0; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1156; feedbackType: None; effectiveEventType: 0> at time 269587.840180
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x170112630; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269588.024871
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x170112a20; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269588.196839
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x170112ab0; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269588.324970
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x174112000; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1156; feedbackType: None; effectiveEventType: 0> at time 269588.444879
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x170112990; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269588.540859
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x170112ea0; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269588.676149
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x1701130e0; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269588.812899
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x1741123f0; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269588.894150
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x170113290; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1156; feedbackType: None; effectiveEventType: 0> at time 269589.012473
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x170113050; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269589.114832
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x170112ea0; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269589.295496
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x174112750; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269589.700663
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x174112a20; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269589.967676
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x1701130e0; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269590.067194
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x170112fc0; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269590.199184
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
activate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 2 -> 1; styleActivationCount: 2 -> 1
playing feedback <_UICustomDiscreteFeedback 0x174112d80; player: 0x1740b48e0; delay: 0.000; effectiveSystemSoundID: 1104; feedbackType: None; effectiveEventType: 0> at time 269590.379329
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=1> with style: TurnOn; activationCount: 1 -> 2; styleActivationCount: 1 -> 2
discovered extensions
deactivate generator <_UIFeedbackKeyboardBehavior: 0x174109b40: prepared=0> with style: TurnOn; activationCount: 0 -> 1; styleActivationCount: 0 -> 1
feedback engine <_UIFeedbackSystemSoundEngine: 0x1740b48e0: state=4, numberOfClients=0, prewarmCount=0, _isSuspended=0> state changed: Running -> Inactive
SFOperationCallBack (<private>, event = Information, results = <private>)
SFBrowserCallBack (node = <SFNode 0x1702e7b80>{domain = AirDrop})
(
Connection to sharingd became invalid
Connection to sharingd became invalid
if we're in the real pre-commit handler we can't actually add any new fences due to CA restriction
if we're in the real pre-commit handler we can't actually add any new fences due to CA restriction
if we're in the real pre-commit handler we can't actually add any new fences due to CA restriction
errors encountered while discovering extensions
Connection to sharingd became invalid
```

There's so much noise in the output. Would be intimidating for developers. We should by minimum somehow make native outputs somehow different in indentation/color/prefix

**Comment by chinmaygarde** [comment]
We are trying to move to a scheme where application logs are forwarded to a socket the tools listen on. I have a PR that is [in progress related to this](https://github.com/flutter/engine/pull/3483). @cbracken should have details on how this is handled on the tools side.

**Comment by xster** [comment]
From #9619, also remove libMobileGestalt messages

**Comment by mit-mit** [comment]
This is causing confusion in IO codelabs. Developers are thinking that all the console output must mean they are doing something wrong. Moving to milestone 3!

**Comment by sethladd** [comment]
I personally agree, the output is scary :/

**Comment by mit-mit** [comment]
I'd like to get this bug assigned; @chinmaygarde @xster @cbracken either of you working on this?

**Comment by cbracken** [comment]
I can take this up (unless @chinmaygarde would prefer to).

**Comment by chinmaygarde** [comment]
@cbracken Go for it!

**Comment by cbracken** [comment]
Fixed by #11969
